api/src/zero/module/xx.clj

```diff
@@ -180,18 +201,26 @@
            (take max)
            (run! release!)))))
 
+(defn print-help
+  [& _]
+  (println description)
+  (println (describe-overrides "baits and targets" "-B" default-baits-and-targets)))
+
 (defn run
   "Reprocess the BAM or CRAM files described by ARGS."
   [& args]
   (try
-    (let [env (zero/throw-or-environment-keyword! (first args))]
-      (apply (case (count args)
-               4 on-hold-some-workflows
+    (let [args-no-overrides (filter #(not (str/starts-with? % "-B")) args)
+          baits-and-targets (apply-overrides default-baits-and-targets "-B" args)
+          env               (zero/throw-or-environment-keyword! (first args-no-overrides))]
+      (apply (case (count args-no-overrides)
+               4 (partial on-hold-some-workflows baits-and-targets)
                3 (partial all/report-status cromwell-label)
                2 release-some-on-hold-workflows
+               1 print-help
                (throw (IllegalArgumentException.
                        "Must specify 2 to 4 arguments.")))
-             env (rest args)))
+             env (rest args-no-overrides)))
```

Comment:
This is why I think having `baits-and-targets` generally be the first argument makes sense. We already do the same thing for `all/report-status` and this lets us very easily `partial` it into the `on-hold-some-workflows` function. This the only place where it really needs to be first, but I went with then being consistent as a default.